server/bin/fxa-content-server.js

```diff
@@ -111,7 +111,7 @@ function makeApp() {
 }
 
 var app,
-    port;
+    port; // eslint-disable-line indent
```

Comment:
> @zaach: _"Ideally we'd have a linter that detected this style of variable declarations and warned about it."_ 

:tada: ZAACH, THIS IS THAT LINTER! :tada: 

---

I think we can use the ESLint [**one-var**](http://eslint.org/docs/rules/one-var.html) rule to forbid the use of multiple declarations using a single `var` or `const`.

```
$ grunt eslint --force
Running "eslint:grunt" (eslint) task


Running "eslint:app" (eslint) task

app/tests/spec/lib/cropper.js
  99:7  error  Split 'var' declarations into multiple statements  one-var

app/tests/spec/lib/metrics.js
  142:7  error  Split 'var' declarations into multiple statements  one-var
  469:7  error  Split 'var' declarations into multiple statements  one-var

app/tests/spec/lib/router.js
  153:7  error  Split 'var' declarations into multiple statements  one-var

app/tests/spec/views/cookies_disabled.js
  19:5  error  Split 'var' declarations into multiple statements  one-var

app/tests/spec/views/form.js
  24:5  error  Split 'var' declarations into multiple statements  one-var

app/tests/spec/views/marketing_snippet.js
  17:5  error  Split 'var' declarations into multiple statements  one-var

app/tests/spec/views/marketing_snippet_ios.js
  19:5  error  Split 'var' declarations into multiple statements  one-var

app/tests/spec/views/mixins/migration-mixin.js
  21:7  error  Split 'var' declarations into multiple statements  one-var

app/tests/spec/views/unexpected_error.js
  17:5  error  Split 'var' declarations into multiple statements  one-var

✖ 10 problems (10 errors, 0 warnings)

Warning: Task "eslint:app" failed. Used --force, continuing.

Running "eslint:tests" (eslint) task

tests/server/routes.js
  13:3  error  Split 'var' declarations into multiple statements  one-var

✖ 1 problem (1 error, 0 warnings)

Warning: Task "eslint:tests" failed. Used --force, continuing.

Running "eslint:scripts" (eslint) task


Running "eslint:server" (eslint) task

server/bin/fxa-content-server.js
  113:1  error  Split 'var' declarations into multiple statements  one-var

✖ 1 problem (1 error, 0 warnings)

Warning: Task "eslint:server" failed. Used --force, continuing.

Done, but with warnings.


Execution Time (2015-08-19 00:07:16 UTC)
loading tasks   3.5s  ▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇ 24%
eslint:grunt    1.1s  ▇▇▇▇▇▇▇▇▇ 8%
eslint:app      7.6s  ▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇ 51%
eslint:tests      2s  ▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇ 13%
eslint:server  681ms  ▇▇▇▇▇ 5%
Total 15.1s
```